packages/core/src/lib/server/createExpressServer.ts

```diff
@@ -48,6 +48,7 @@ const addApolloServer = async ({
     app: server,
     path: config.graphql?.path || '/api/graphql',
     cors: false,
+    bodyParserConfig: config.graphql?.bodyParser === undefined ? true : config.graphql?.bodyParser,
```

Comment:
Is `true` equivalent to the default? and if so, why doesn't just `config.graphql?.bodyParser` work?

Reply:
`true` is equivalent to the `express` defaults, and anything `!== false` is explicitly provided.

Reference https://github.com/apollographql/apollo-server/blob/e5144f160f4d49f9651473ff092c7ae11d433d82/packages/apollo-server-express/src/ApolloServer.ts#L128-L132

That said, reading https://github.com/expressjs/body-parser/blob/1f6f58e1f8dc222f2b6cfc7eb3a3bf5145ff2b56/lib/types/json.js#L51 - it doesn't matter anyway.